main/main.go

```diff
@@ -143,6 +146,11 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err := initSeedGen(context.TODO(), mgr.GetClient(), &setupLog); err != nil {
```

Comment:
Perhaps we can create a context variable and pass the same context to both the `initIBU` and `initSeedGen` functions?